recipes/reduce/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+mkdir -p build
+cd build
+cmake ${SRC_DIR} ${CMAKE_ARGS}
+make
+make install
```

Comment:
How does it know where to install ?
I think you need something like CMAKE_INSTALL_PREFIX=$PREFIX

Reply:
@martin-g 

`${CMAKE_ARGS}` should include `-DCMAKE_INSTALL_PREFIX=$PREFIX` implicitly.

Reply:
@martin-g 
Should I change `${CMAKE_ARGS}` to `-DCMAKE_INSTALL_PREFIX=$PREFIX` explicitly?

Reply:
Fixed. 

https://github.com/bioconda/bioconda-recipes/pull/49610/commits/ca4c71cc4ba8e7cc668e08d45769fa633f66b9fe